detection_rules/devtools.py

```diff
@@ -1211,6 +1218,12 @@ def show_latest_compatible_version(package: str, stack_version: str) -> None:
         click.echo(f"Error loading integrations manifests: {str(e)}")
         return
 
+    try:
+        stack_version = Version.parse(stack_version)
+    except Exception as e:
+        click.echo(f"Error parsing stack version: {str(e)}")
+        return
+
```

Comment:
You can probably use the `isvalid` method to check if the string is a valid server version.